pwiz_tools/Skyline/Model/Proteome/ProteinAssociation.cs

```diff
@@ -175,10 +175,12 @@ public void UseBackgroundProteome(BackgroundProteome backgroundProteome, Func<Fa
             
             Assume.IsTrue(localResults.ProteinsMapped + localResults.ProteinsUnmapped > 0);
 
+            var distinctPeptideDocNodes = _peptideToPath.SelectMany(kvp => kvp.Value).Distinct().ToList();
+            int distinctTargetPeptideCount = distinctPeptideDocNodes.Where(p => !p.IsDecoy).Select(p => p.Peptide.Target).Distinct().Count();

```

Comment:
I would write this as:
```
int distinctTargetPeptideCount = _peptideToPath.SelectMany(kvp => kvp.Value)
    .Where(p => !p.IsDecoy)
    .Select(p => p.Peptide.Target).Distinct().Count();
```
This avoids the calling "Distinct" on the collection of PeptideDocNode's on the previous line, which is redundant because you eventually call "Distinct" on the Target's anyway.
(PeptideDocNode.GetHashCode and PeptideDocNode.Equals are both big slow functions so it's best to avoid calling them if you can)

Reply:
I think I had it that way to speed up interactive debugging and forgot to change it back. Thanks for catching that.